providers/cloudflare/cloudflareProvider.go

```diff
@@ -638,3 +680,19 @@ func (c *cloudflareProvider) EnsureDomainExists(domain string) error {
 	fmt.Printf("Added zone for %s to Cloudflare account: %s\n", domain, id)
 	return err
 }
+
+// PrepareCloudflareWorkers creates Cloudflare Workers required for CF_WORKER_ROUTE tests.
```

Comment:
I feel like this should move to the integrationTests source to prevent bringing `testing` into the production build.